linode/manager
# chore: Storybook Upgrade and Enhancements

## Description 📝

- Storybook dependency Upgrade and Enhancements 

## Major Changes 🔄
- Removed `@mdx-js/loader` because we don't need to directly depend on it
- Removed `@storybook/addon-links` because it seems like we don't really use it or need it
- Updates all Storybook dependencies to `7.0.18` (latest at the time of this PR) 🎉
- Adds `@storybook/addon-measure` to give us a new useful feature 🚀
  - Maybe our UI/UX will find this helpful 🤞🏼
- Adds `@storybook/addon-actions` to our addons array
  - We use this in one story (Dialog) but we never properly setup the addon 🔧 It will now show as a tab beside the "Controls"

## Preview 📷

> **Note** Demos the new addon `@storybook/addon-measure`

https://github.com/linode/manager/assets/115251059/ba3e1be9-423d-4cd4-8eed-e21061f3e4a0

## How to test 🧪

- Checkout this PR
- Run `yarn` for good measure so your dependencies are up to date
- Run `yarn storybook`
- Verify Storybook works as you'd expect and all addons/features work as you'd expect
